parser/CMakeLists.txt

```diff
@@ -14,15 +14,17 @@ add_executable(CodeCompass_parser
 set_target_properties(CodeCompass_parser
   PROPERTIES ENABLE_EXPORTS 1)
 
-target_link_libraries(CodeCompass_parser
-  util
-  model
+find_boost_library(
   boost_thread
   boost_filesystem
   boost_system
   boost_program_options
-  boost_log
-  ${ODB_LIBRARIES}    
+  boost_log)
```

Comment:
Maybe the parameters here should be just `filesystem log program_options system thread`? (See my other comment in the newly introduced function.)

Reply:
Updated accordingly.